packages/terra-application-header-layout/src/ApplicationHeaderLayout.module.scss

```diff
@@ -16,7 +16,7 @@
     align-items: center;
     display: flex;
     height: 100%;
-    overflow: hidden;
+    overflow: visible;
```

Comment:
Do you remember why this change was made?

Reply:
@neilpfeiffer made the change, so I am pinging him for the question.

Reply:
My thought is this is to allow the box shadow to display? cc/ @neilpfeiffer

Reply:
most likely that is correct.